tests/authenticator_tests/testing/mocks/mocks/redis_sessions.py

```diff
@@ -9,8 +9,20 @@
 
 class RedisSessions(object):
     @staticmethod
-    def set(key, val):
-        session_data[key] = val
+    def set(
+        self,
```

Comment:
`self` with a staticmethod feels wrong? 👀  Possibly I/we forgot to remove @staticmethod? does that break it again?